packages/api-bindings/mocks/package.json

```diff
@@ -1,4 +1,5 @@
 {
```

Comment:
these extra  dirs and `package.json` are autogenerated, we could consider adding to .gitignore?

Reply:
nope, can't ignore them as they should be pushed to the `npm` registry so bundlers can find properly imports
Like technically you can ignore and than make sure you still push them to the registry but it's probably not worth to add one layer of possible mistakes